packaging/debian-sid/rules

```diff
@@ -142,18 +143,18 @@ override_dh_auto_build:
 	# XXX: once dh-golang understands go build tags this would not be needed
 	find _build/src/$(DH_GOPKG)/secboot/ -name "*.go" | grep -Ev '(encrypt\.go|secboot_dummy\.go|secboot\.go|encrypt_dummy\.go)' | xargs rm -f
 	# and build
-	dh_auto_build -- $(BUILDFLAGS) -tags "$(TAGS)" $(GCCGOFLAGS)
+	GO111MODULE=off dh_auto_build -- $(BUILDFLAGS) -tags "$(TAGS)" $(GCCGOFLAGS)
```

Comment:
I think this deserves a comment that we can't use modules on debian sid so we build just using the vendor dir that was created with `go -mod=vendor ...` previously